tools/autograd/templates/python_torch_functions.cpp

```diff
@@ -465,6 +467,99 @@ static PyObject * THPVariable_get_device(PyObject* self_, PyObject* args, PyObje
   END_HANDLE_TH_ERRORS
 }
 
+static PyObject * THPVariable_frombuffer(PyObject* self_, PyObject* args, PyObject* kwargs)
+{
+  HANDLE_TH_ERRORS
+  static PythonArgParser parser({
+    "frombuffer(PyObject* buffer, ScalarType dtype=None, int64_t count=-1, int64_t offset=0, *, bool requires_grad=False)",
```

Comment:
What would you think of making everything after "buffer" kwarg-only? This would be more consistent with other PyTorch operations that take "tensor options" (like dtype, device, requires_grad...) and the Python Array API's style

Reply:
I think that is probably a good idea. Another thing that would probably be better is to make `dtype` a required argument. What do you think? Since we are blindly believing in the specified `dtype`, maybe we could make the user explicitly specify it so as not to have any (less) surprising behaviors.

Reply:
Making it required for now seems OK